sparse/etc/acdbdata

```diff
@@ -0,0 +1 @@
+/system/etc/acdbdata
```

Comment:
perfect, so you could rename the commit to "[dcd] Fix firmware find path for sound dsp"